frontend/src/components/panels/home_panel/home_panel_components/IconRow.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import LocationMarker from '../../../helpers/locationMarker';
+
+const IconRow = (props) => (
```

Comment:
Per other comment `({ text })` would be a clearer contract to the developer as to what props the component uses.